sce.py

```diff
@@ -1,19 +1,62 @@
 import argparse
+from subprocess import check_output, PIPE
 from tools.setup import SceSetupTool
 
+
+test_commands = {
+    'Core-v4': ['npm run api-test --prefix ./Core-v4', 'npm run lint --prefix ./Core-v4', \
+        'npm run frontend-test --prefix ./Core-v4', 'npm run build --prefix ./Core-v4'],
+    'sce-rpc': ['npm run test --prefix ./sce-rpc', 'npm run lint --prefix ./sce-rpc', \
+        'pylint * --prefix ./sce-rpc'],
+    'SCE-discord-bot': ['npm run lint --prefix ./SCE-discord-bot'],
+    'dev': ['pylint * --ignore=Core-v4,sce-rpc,SCE-discord-bot']
+}
+
 setup = SceSetupTool()
 parser = argparse.ArgumentParser()
 
-parser.add_argument('--setup', dest='setup', action='store_true')
+parser.add_argument('command', help='SCE Command')
+parser.add_argument('-p', '--project', nargs='*', help='Docker Container Name')
 args = parser.parse_args()
 
-if args.setup:
+if args.command in 'setup':
     setup.check_os()
     setup.check_docker()
     setup.check_mongo()
 
     setup.setup_rpc()
     setup.setup_core_v4()
     setup.setup_discord_bot()
-else:
-    print('did nothing wrong')
+
+def run_test(test_name, command):
+    num_dots = 37 - len(test_name)
+    dots = "."
+    dots = dots * num_dots
+    dots_passed = dots + "passed"
+    dots_failed = dots + "failed"
+    try:
+        check_output(command, shell=True, stderr=PIPE)
+        print('     ' + test_name + dots_passed)
+    except: # pylint: disable=bare-except
+        print('     ' + test_name + dots_failed)
+
+
+def project_tests(proj):
```

Comment:
```suggestion
def project_tests(project):
```

Reply:
I originally changed it from project to proj because the pylint docked off some points for having it being called project for some reason

Reply:
still change it back?

Reply:
lets keep it for now, but open an issue for someone to investigate why pylint is being like that